adcc/workflow.py

```diff
@@ -495,3 +509,73 @@ def setup_solver_printing(solmethod_name, matrix, kind, default_print,
         def inner_callback(state, identifier):
             default_print(state, identifier, output)
         return inner_callback
+
+
+def setup_environment(matrix, environment):
+    """
+    Setup environment matrix terms and/or energy corrections.
+    Internal function called from run_adc.
+    """
+    valid_envs = ["ptss", "ptlr", "linear_response"]
+    hf = matrix.reference_state
+    if hf.environment and environment is None:
+        raise InputError(
+            "Environment found in reference state, but no environment"
+            " configuration specified. Please select from the following"
+            f" schemes: {valid_envs}."
```

Comment:
But `False`is also possible, which you don't tell the user here. That's confusing.

Reply:
Good point!

Reply:
Done.